app/javascript/components/ExpressDonation/ExpressDonation.js

```diff
@@ -80,6 +80,11 @@ export class ExpressDonation extends Component {
 
   submit() {
     const data = this.oneClickData();
+    ee.emit(
+      'fundraiser:transaction_submitted',
+      data.payment,
```

Comment:
Data here could be null (`this.oneClickData()` can return null) so we also need to wrap that in the `if (data)` block.